src/glpk.js.c

```diff
@@ -12,11 +12,13 @@ int solve_lp(glp_prob *P, int msg_lev) {
 	return glp_simplex(P, &parm);
 }
 
-int solve_mip(glp_prob *P, int msg_lev) {
+int solve_mip(glp_prob *P, int msg_lev, float mip_gap, float tm_lim) {
```

Comment:
I think the ideal behavior here would be to pass some sort of struct that that can take the exact same values as `glp_iocp` below. I am not sure how to do that though.